src/Agent/NewRelic/Agent/Core/JsonConverters/EventAttributesJsonConverter.cs

```diff
@@ -83,6 +86,15 @@ public override void WriteJson(JsonWriter writer, IEnumerable<KeyValuePair<strin
                 {
                     writer.WriteValue((byte)kvp.Value);
                 }
+                else if (kvp.Value is StackTrace)
+                {
+                    using (var configurationSubscription = new ConfigurationSubscriber())
```

Comment:
I'm curious what would happen if we made a single static config subscriber.... It would be a better pattern vs creating/destroying a subscriber whenever access is needed